.maintain/frame-weight-template.hbs

```diff
@@ -29,7 +31,7 @@ pub trait WeightInfo {
 	{{/each}}
 }
 
-/// Weights for {{pallet}} using the Data Avaiability node and recommended hardware.
+/// Weights for `{{pallet}}` using the Substrate node and recommended hardware.
```

Comment:
Should we use Data Availability instead of Substrate in the template?

Reply:
Yeah this is just a copy paste😅